src/Assets/styleController.ts

```diff
@@ -73,6 +73,7 @@ function generateMuiOverides(theme: ITheme) {
   const darkTerraCottaColor = theme.secondaryColor;
   const blackColor = '#2A2B2A';
   const midBlue = theme.midBlueColor;
+  const lavenderColor = '#ECDEED';
```

Comment:
Referenced this [here](https://github.com/Gary-Community-Ventures/benefits-calculator/pull/791/files/6ccd6ea24e623d2ec9ad78e1007c469333487b5d#r1418129878), but could we add this to the style controller with a name that describes what it does.